### PR TITLE
Feature/sshkey

### PR DESCRIPTION
**Pull request by whispyy**

### CODE REVIEW

File: dist/ovh-api-services.js [hunk 1]
```diff
@@ -5810,12 +5810,42 @@ angular.module("ovh-api-services").service("OvhApiMeSshKey", ["$injector", funct
...
+            return response.resource;
+        }
+    };
+
+    var resource = $resource("/me/sshKey/:keyName", { keyName: "@keyName" }, {
+        query: { method: "GET", cache: queryCache, isArray: true },
+        get: { method: "GET", cache: cache },
+        create: { method: "POSt", interceptor: interceptor },
+        edit: { method: "PUT", interceptor: interceptor },
+        "delete": { method: "DELETE", interceptor: interceptor }
```
Comment (jleveugle) [inline dist/ovh-api-services.js]: -- double quote ?

It's not a keyword here, it's a JSON ^.^

File: dist/ovh-api-services.js [hunk 2]
```diff
@@ -5810,12 +5810,42 @@ angular.module("ovh-api-services").service("OvhApiMeSshKey", ["$injector", funct
     };
 }]);
 
-angular.module("ovh-api-services").service("OvhApiMeSshKeyLexi", ["$injector", "$resource", function ($injector, $resource) {
+angular.module("ovh-api-services").service("OvhApiMeSshKeyLexi", ["$resource", "$cacheFactory", function ($resource, $cacheFactory) {
```
Comment (jleveugle) [inline dist/ovh-api-services.js]: No need to specify injections, this projet has a task to do that : `ngAnnotate`

File: src/me/sshkey/me-sshkey.lexi.service.js [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
+            cache.remove(response.config.url);
+            queryCache.removeAll();
+            return response.resource;
+        }
+    };
+
+    var resource = $resource("/me/sshKey/:keyName", { keyName: "@keyName" }, {
+        query: { method: "GET", cache: queryCache, isArray: true },
+        get: { method: "GET", cache: cache },
+        create: { method: "POSt", interceptor: interceptor },
```
Comment (Netnix) [inline src/me/sshkey/me-sshkey.lexi.service.js]: typo:
POSt -> POST

File: dist/ovh-api-services.js (hunk 1 above)
Comment (Netnix) [reply dist/ovh-api-services.js]: It is a keyword. All the delete in ovh-api-services have double quotes.

File: src/me/sshkey/me-sshkey.lexi.service.js (hunk 3 above)
Comment (whispyy) [reply src/me/sshkey/me-sshkey.lexi.service.js]: oh yes sorry for that !

**Pull request merged**